packages/core/src/lib/Uuid.ts

```diff
@@ -2,6 +2,17 @@ import { randomUUID } from "crypto"
 
 export type Uuid<T> = string & { __brand: T }
 
+// permissive uuid regex that allows non-standard uuids
```

Comment:
doesnt look very permissive from where I'm sitting?

Reply:
Permissive in that it accepts any hexadecimal string as long as it looks like a uuid. The canonical regex looks like this https://github.com/uuidjs/uuid/blob/main/src/regex.js